activitysim/abm/test/test_misc/configs_test_misc/trip_departure_choice.yaml

```diff
@@ -0,0 +1,20 @@
+#METADATA:
```

Comment:
What's "METADATA" for here?  Seems like nothing, since it's commented out?  We should not add new code/tests with commented out old code.